src/components/Common/SubLabel.css

```diff
@@ -7,6 +7,6 @@
   font-style: normal;
   font-weight: 400;
   font-size: 15px;
```

Comment:
I think we can remove font-weight, style, color, and size if we are using Text's css

Reply:
This is overriding the Markdown component styles 😞